text/0016-internationalization.md

```diff
@@ -0,0 +1,117 @@
+
+# Internationalization
+
+- Start Date: 2019/09/26
+- RFC PR: https://github.com/openmrs/openmrs-rfc-frontend/pull/17
+	
+
+## Definitions
+
+Localization refers to the adaptation of a product, application or document content to meet the language, cultural and other requirements of a specific target market (a locale).
+
+It is sometimes written as l10n, where 10 is the number of letters between l and n.
+
+  
+
+Internationalization is the design and development of a product, application or document content that enables easy localization for target audiences that vary in culture, region, or language. It is often written i18n, where 18 is the number of letters between i and n.
+
+  
+
+Transifex is a cloud-based localization platform built to help you manage the translation and localization of your app, website, video subtitles, and more. It acts as a repository for your content (think GitHub for translation) and includes tools for developers to get that content into Transifex automatically. Transifex also provides translators a web interface to submit translations.
+
+  
+
+## Decision, including impact on distributions
+
+OpenMRS is implemented throughout the world and is used by speakers of many different languages. As not every user speaks English as a primary language, we will:
+
+1.  Provide an ESM module to translate content in the UI to the user’s specified language.
```

Comment:
If I understand correctly, we have translation is not as its own module, but rather integrated into every module. Let's update the text of this document?